wasm.md

```diff
@@ -250,18 +266,26 @@ Comparisons consume two operands and produce a bool, which is an `i32` value.
 
 ```k
     syntax RelOp ::= IRelOp
- //                | FRelOp
  // -----------------------
 
     syntax Instr ::= "(" IValType "." IRelOp ")" | "(" IValType "." IRelOp Instr Instr ")" | IValType "." IRelOp Int   Int
- //                  "(" FValType "." FRelOp ")" | "(" IValType "." FRelOp Instr Instr ")" | IValType "." FRelOp Float Float
  // ------------------------------------------------------------------------------------------------------------------------
     rule <k> ( ITYPE . ROP:IRelOp I:Instr I':Instr ) => I ~> I' ~> ( ITYPE . ROP ) ... </k>
 
     rule <k> ( ITYPE . ROP:IRelOp ) => ITYPE . ROP C1 C2 ... </k>
          <valstack> < ITYPE > C2 : < ITYPE > C1 : VALSTACK => VALSTACK  </valstack>
 ```
 
+**TODO**: Unimplemented.
+
+```k
+    syntax RelOp ::= FRelOp
+ // -----------------------
+
+    syntax Instr ::= "(" FValType "." FRelOp ")" | "(" IValType "." FRelOp Instr Instr ")" | IValType "." FRelOp Float Float
```

Comment:
Should be IValType in the 2nd and 3rd sorts

Reply:
fixed